### ISSUE TITLE
[Feature] Another tray icon state for when busy doing cleaning

### ISSUE THREAD

**Issue by vassilyzaitsev** [henrypp/memreduct#188]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

3.4

### Problem you are trying to solve

I don't know exactly when the app is doing something when I trigger the cleaning from the tray icon.

### Suggested solution

To add another color to the appearance options for "busy".

**Comment by henrypp** [comment]
>To add another color to the appearance options for "busy".

Like what?

**Comment by vassilyzaitsev** [comment]
You mean for the default? Maybe yellow.

Just to be clear, this is what I meant:

https://i.imgur.com/ClB2hTl.png

**Comment by henrypp** [comment]
And where is the logic? Display danger as yellow is, i don't know, idiotic, it should be red, like traffic lights, red is danger, yellow is warning, all know this.

so, i do not know where is the problem to change option by yourself.

**Comment by vassilyzaitsev** [comment]
No. I'm sorry I wasn't clear. I'd like to suggest that an option to add another color option for when the app is busy cleaning the memory. 

Because at the moment, when you click on the icon tray for it to clean the memory. The color doesn't change, so you don't know when it starts nor when it finishes the cleaning routine.

![image](https://user-images.githubusercontent.com/35409978/234940683-23dba44a-3597-434f-baba-0a3416d7dfdd.png)

The color yellow doesn't matter. It's just for everyone to know when Memreduct is doing the cleaning.
